daml-lf/kvutils/README.md

```diff
@@ -0,0 +1,11 @@
+The KV Daml-LF library
+======================
+
+This package contains LF utilities specifically tailored for the KV
+integration kit. This library does not define a stable public API, and
+should not be used outside the internal of the KV integration kit.
+Breaking changes to this library will be made without notice.
+
+Any modification to this library should be approved by at least 
```

Comment:
Instead of adding a note here, let's modify the `CODEOWNERS` file and add the relevant people there instead to have them automatically notified when this module is being modified.

Reply:
:+1: